src/page/template/content/preferences-options.htm

```diff
@@ -1,77 +1,3 @@
-          <input type="radio" id="preferences-options-notification-obfuscate" name="preferences-options-notification" value="obfuscate" data-bind="checked: optionNotifications">
-          <label class="accent-text" for="preferences-options-notification-obfuscate">
-            <span class="preferences-label" data-bind="text: t('preferencesOptionsNotificationsObfuscate')"></span>
-          </label>
-          <input type="radio" id="preferences-options-notification-none" name="preferences-options-notification" value="none" data-bind="checked: optionNotifications">
-          <label class="accent-text" for="preferences-options-notification-none">
-            <span class="preferences-label" data-bind="text: t('preferencesOptionsNotificationsNone')"></span>
-          </label>
-        </div>
-      </section>
-
-      <section class="preferences-section">
-        <header class="preferences-header" data-bind="text: t('preferencesOptionsPopular')"></header>
-        <div class="preferences-option">
-          <div class="preferences-option-icon checkbox accent-text" data-bind="attr: {'data-uie-value': optionDarkMode}" data-uie-name="status-preference-use-dark-mode">
-            <input type="checkbox" id="use-dark-mode-checkbox" data-bind="checked: optionDarkMode">
-            <label class="preferences-options-checkbox-label" for="use-dark-mode-checkbox" data-bind="text: t('preferencesOptionsUseDarkMode')"></label>
-          </div>
-        </div>
-        <div class="preferences-option">
-          <div class="preferences-option-icon checkbox accent-text" data-bind="attr: {'data-uie-value': optionReplaceInlineEmoji}" data-uie-name="status-preference-emoji-replace">
-            <input type="checkbox" id="replace-inline-emoji-checkbox" data-bind="checked: optionReplaceInlineEmoji">
-            <label class="preferences-options-checkbox-label" for="replace-inline-emoji-checkbox" data-bind="text: t('preferencesOptionsEmojiReplaceCheckbox')"></label>
-          </div>
-        </div>
-        <div class="preferences-detail preferences-detail-emoji" data-bind="html: t('preferencesOptionsEmojiReplaceDetail', {}, {'icon': '<span class=\'font-size-xs icon-emoji\'></span>'})"></div>
-        <div class="preferences-option preferences-options-multiple">
-          <div class="preferences-option-icon checkbox accent-text" data-bind="attr: {'data-uie-value': optionSendPreviews}" data-uie-name="status-preference-previews-send">
-            <input type="checkbox" id="send-preview-checkbox" data-bind="checked: optionSendPreviews">
-            <label class="preferences-options-checkbox-label" for="send-preview-checkbox" data-bind="text: t('preferencesOptionsPreviewsSendCheckbox')"></label>
-          </div>
-        </div>
-        <div class="preferences-detail" data-bind="text: t('preferencesOptionsPreviewsSendDetail')"></div>
-      </section>
-    <!-- /ko -->
-
-  </div>
+<div id="preferences-options" class="preferences-page preferences-options">
```

Comment:
Is this template still used? Can't we move this html chunk to the `OptionPerferences` component and render the component directly in the parent component?

Reply:
That's a pretty good point, I should also look into changing it for the other already refactored page.